metrics.yaml

```diff
@@ -218,3 +218,17 @@ setting_event:
         description: The location that this setting was changed from
         type: string
     expires: never
+telemetry_event:
```

Comment:
For consistencies sake, it should probably be called installId or telemetryInstallId everywhere. The doc I wrote and the preference are called installId. Unless that conflicts with something else already?